### ISSUE TITLE
Svelte 5 support?

### ISSUE THREAD

**Issue by ndom91** [janosh/svelte-multiselect#281]
## Repro

- Svelte 5 Repro broken: https://stackblitz.com/edit/github-y3fhqr?file=package.json
- Looks like the event attributes must be functions, not strings, i.e.
    - Not `ondrop="return false"`, but `ondrop={() => false}`

![image](https://github.com/janosh/svelte-multiselect/assets/7415984/7d84ac17-8be1-4fbd-bedb-4dbbb42b542a)

## Additional context

Are there any plans for Svelte 5 updates yet? I know its early, but yeah figured I'd ask :blush:

**Comment by janosh** [comment]
@ndom91 Svelte v5 support is definitely coming!

I remember there was a reason for having `ondrop="return false"` over `ondrop={() => false}` but maybe that's no longer the case with Svelte 5. If so, easy fix.
The main blocker right now is `mdsvexamples` used extensively in the docs. Just upgraded to `5.0.0-next.35` and still get `vite` compilation errors:

```js
    at option (/Users/janosh/dev/svelte-multiselect/src/site/Examples.svx___mdsvexample___2.svelte:37:49)
    at Module.slot (/Users/janosh/dev/svelte-multiselect/node_modules/.pnpm/svelte@5.0.0-next.35/node_modules/svelte/src/internal/server/index.js:440:3)
    at $$render_inner (/Users/janosh/dev/svelte-multiselect/src/lib/MultiSelect.svelte:762:27)
    at MultiSelect (/Users/janosh/dev/svelte-multiselect/src/lib/MultiSelect.svelte:885:3)
    at Examples_svx___mdsvexample___2 (/Users/janosh/dev/svelte-multiselect/src/site/Examples.svx___mdsvexample___2.svelte:26:35)
    at /Users/janosh/dev/svelte-multiselect/src/site/Examples.svx:189:48
    at Module.slot (/Users/janosh/dev/svelte-multiselect/node_modules/.pnpm/svelte@5.0.0-next.35/node_modules/svelte/src/internal/server/index.js:437:4)
    at example (/Users/janosh/dev/svelte-multiselect/src/site/Examples.svx:184:11)
    at Module.slot (/Users/janosh/dev/svelte-multiselect/node_modules/.pnpm/svelte@5.0.0-next.35/node_modules/svelte/src/internal/server/index.js:440:3)
    at CodeExample (/Users/janosh/dev/svelte-multiselect/node_modules/.pnpm/svelte-zoo@0.4.9_vite@5.0.11/node_modules/svelte-zoo/dist/CodeExample.svelte:70:25)
11:43:02 AM [vite] warning: Example src not found for /Users/janosh/dev/svelte-multiselect/___mdsvexample___0.svelte
```

Feel free to chime in at https://github.com/mattjennings/mdsvexamples/issues/30 if you think now's the time to address these.

**Comment by rchavezj** [comment]
Svelte 5 worked for me.

**Comment by davidlbangs** [comment]
Did you have trouble with bind?  Two different bind systems mean that the control's selection is not communicated to a parent component using Svelte 5.  How did you get around that?

**Comment by janosh** [comment]
busy with other stuff lately but i just resolved the merge conflicts and fixed a few failing unit tests in #295. it's getting late here and 13 tests are still failing. can try to take another swing at fixing them tomorrow. if anyone wants to take a look in the meantime, be my guest!
